src_c/event.c

```diff
@@ -364,6 +364,8 @@ _pg_name_from_eventtype(int type)
         case SDL_NOEVENT:
             return "NoEvent";
 #if IS_SDLv2
+        case SDL_WINDOWEVENT:
+            return "WindowEvent";
```

Comment:
This should probably be extended to handle all of the cases that are missing, instead of just this one, as the rest of the PR appears to be aware of said other cases.

Reply:
The other event ids already exist here. Refer to lines 328-335 and 385-392.